controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTriggerTest.java

```diff
@@ -118,7 +120,6 @@ public void deploymentsSpecWithDelays() {
 
         ApplicationPackage applicationPackage = new ApplicationPackageBuilder()
                 .environment(Environment.prod)
-                .delay(Duration.ofSeconds(30))
```

Comment:
This should still be supported/tested.

Reply:
Why? The deployment spec documentation refers to delays as something you add _between_ deployments for a chance to discover errors before they propagate to more zones. 

Deployments in test and staging don't live on after their test jobs, and so won't really offer this possibility, and I don't think anyone has alerting set up for deployments in those zones?

If you want to delay production altogether, isn't that when you use `UpgradePolicy.conservative`?

Reply:
Right, I'll re-introduce this.